ci/redmine_bugzilla.py

```diff
@@ -149,12 +150,22 @@ def main():
                         links_issues_record += 'Redmine #%s -> Bugzilla %s, but Bugzilla %s does not ' \
                                            'link back\n' % (issue.id, bug.id, bug.id)
 
+    start_time = time.time()
+    redmine_issues_looked_up = 0
     for bug in bugzilla_bugs:
         for external_bug in bug.external_bugs:
                 if external_bug['type']['description'] == 'Pulp Redmine':
                     add_cc_list_to_bugzilla_bug(bug)
                     issue_id = external_bug['ext_bz_bug_id']
-                    issue = redmine.issue.get(issue_id)
+                    try:
+                        redmine_issues_looked_up += 1
+                        issue = redmine.issue.get(issue_id)
+                    except:
+                        end_time = time.time()
+                        time_to_query_before_failure = start_time - end_time
+                        print 'Total time to query %d redmine issues: %f' %\
```

Comment:
Yes, that's true. Even so, I'd advocate for using `from __future__ import print_function`. One benefit of doing so is that the same function will be used regardless of whether Python 2.7 or 3.x is used to execute the script. In a simple case like `print('Hi')`, it doesn't really matter. In a more complex case like `print('Hi', end='')`, it does matter.